protocol/avdl/chat_local.avdl

```diff
@@ -50,8 +51,20 @@ protocol chatLocal {
     union { null, chat1.Pagination } pagination;
   }
 
+  record MessageSelector {
+    array<chat1.MessageType> MessageTypes;
+
+    union { null, Time } After;
+    union { null, Time } Before;
+    bool onlyNew;
+    int limitNumber;
+  }
+
   chat1.InboxView getInboxLocal(union { null, chat1.Pagination } pagination);
   ThreadView getThreadLocal(chat1.ConversationID conversationID, union { null, chat1.Pagination } pagination);
   void postLocal(chat1.ConversationID conversationID, MessagePlaintext messagePlaintext);
-  void newConversationLocal(chat1.ConversationIDTriple conversationTriple);
+  chat1.ConversationID newConversationLocal(chat1.ConversationIDTriple conversationTriple);
+
+  chat1.ConversationID getOrCreateTextConversationLocal(string tlfName, string topicName, chat1.TopicType topicType);
```

Comment:
We have no mechanism in place to ensure that this will be unique resolution.  You can definitely get multiple conversations that have the same `(tlf, type, id)` triple.

Also, can you reorder the arguments to be `(tlf, type, name)`?